rls/src/main/java/io/grpc/rls/CachingRlsLbClient.java

```diff
@@ -969,7 +1003,22 @@ private PickResult useFallback(PickSubchannelArgs args) {
       if (picker == null) {
         return PickResult.withNoResult();
       }
-      return picker.pickSubchannel(args);
+      PickResult pickResult = picker.pickSubchannel(args);
+      // TODO: include the grpc.target label once target is available here.
+      helper.getMetricRecorder().addLongCounter(DEFAULT_TARGET_PICKS_COUNTER, 1,
+          Lists.newArrayList("", rlsConfig.lookupService(), fallbackChildPolicyWrapper.getTarget(),
+              determineMetricsPickResult(pickResult)), Lists.newArrayList());
+      return pickResult;
+    }
+
+    private String determineMetricsPickResult(PickResult pickResult) {
+      if (pickResult.getStatus().equals(Status.OK)) {
```

Comment:
What you really want here is `getSubchannel() != null`, to mean "complete." But it is possible for the status to be OK and there not to be a subchannel, in which case the result is "buffer." It seems we should not record a metric at all in that case.